website/content/docs/secrets/ssh/signed-ssh-certificates.mdx

```diff
@@ -1,15 +1,8 @@
 ---
 layout: docs
-page_title: Signed SSH Certificates - SSH - Secrets Engines
+page_title: Signed SSH certificates
 description: >-
-  The signed SSH certificates is the simplest and most powerful in terms of
-
-  setup complexity and in terms of being platform agnostic. When using this
-
-  type, an SSH CA signing key is generated or configured at the secrets engine's
-  mount.
-
-  This key will be used to sign other SSH keys.
+  Generate an SSH CA signing key with the SSH secrets engine to securely connect to a remote machine.
```

Comment:
This removes the emphasis that this is our desired method of configuring SSH.  This text needs to be maintained somewhere in the doc.

Reply:
The original description looks like a copy of the first paragraph on the doc page. 

![image](https://github.com/user-attachments/assets/f79b5226-6936-4f20-95fa-c8011d7d404e)

What we're doing with the `description` is to improve the discoverability of the page. 

Since Google typically displays only 155–160 characters in search results, we want to keep the description to around 150 characters long. Ideally, the description has keywords & search terms related to the page.

Reply:
[Done](https://github.com/hashicorp/vault/pull/29116/commits/4ef334ad5a9fae249316ba74d5144484381ce32c)